### PR TITLE
Docker compose with nginx reverse proxy

### PR DESCRIPTION
**Pull request by TechwizEE**
Adding documentation for a self-host option using Docker Compose with NGINX Reverse Proxy providing HTTPS.  Also added a small FAQ section providing references the "merged budget" issues.  

This new self-host option is in response to the "merged budge" issue encountered by several Actual users.  The solution provides HTTPS access to ActualServer which enables a browser to honor the Cross-Origin-Opener-Policy (COOP) specified by ActualServer (enables `SharedArrayBuffer`).

I welcome feedback, critiques, recommendations.  The documentation I write is usually quite technical.  I've worked to make this contribution more narrative and (hopefully) accessible by a wide variety of users.

Thread over in Discord: https://discord.com/channels/937901803608096828/1063869043465007236

### CODE REVIEW

File: docs/FAQ.md [hunk 1]
```diff
@@ -58,3 +58,16 @@ title: 'Frequently Asked Questions'
   In that case, you need to keep some negative dollars from touching positive dollars by sticking them
   in a category.  Actual's default assumption is that you can pay off any card in full at any time and
   not touch your budget to do it. (Credit: evequefou)
+
+  **Q.** *I'm self-hosting ActualServer and have created several budgets.
```
Comment (MatissJanis) [inline docs/FAQ.md:62]: Would you mind removing this FAQ bulletpoint? IMO it's an issue we should address with a fix rather than documenting a workaround.

**Review by MatissJanis — COMMENTED** [review]
👋 The docker + nginx page looks sweet! Good job.

File: docs/FAQ.md (hunk 1 above)
Comment (TechwizEE) [reply docs/FAQ.md]: Certainly.  I agree implementing a code fix is better than "fixing" the bug with a work-around.

**Comment by TechwizEE** [comment]
> 👋 The docker + nginx page looks sweet! Good job.

Thanks for taking the time to review!  My addition isn't exactly short.

**Comment by TechwizEE** [comment]
Quick check-in.  Is this documentation still something good to add to Actual?  I'm open either way.    Would be interested in any feedback.

**Comment by rich-howell** [comment]
@TechwizEE thanks for this. Merging

**Pull request merged**